icecream.py

```diff
@@ -104,6 +104,31 @@ def update_ice_cream(id):
 def delete_flavor(id):
     del icecreams[id]
     return '', HTTP_204_NO_CONTENT
+    
+    
+######################################################################
+# RETRIEVE ice-creams based on popularity
+# http://localhost:5000/ice-creams?popularity=4 fetches all the
+# ice creams with popularity greater equal to 4.0
+######################################################################
+@app.route('/ice-cream/', methods=['GET'])
```

Comment:
can we change this route to something besides the default `/ice-cream/`?

maybe something like `/ice-cream/popular`?